stock_picking_invoicing_unified/wizard/stock_invoice_onshipping.py

```diff
@@ -11,19 +11,41 @@ class StockInvoiceOnshipping(models.TransientModel):
 
     _inherit = 'stock.invoice.onshipping'
 
+    @api.model
+    def _default_journal(self, journal_type):
```

Comment:
Defaults are not needed, because they get assigned on init thanks to onchange_group method.